### ISSUE TITLE
lukko fails to cross-compile, breaking cabal-install downstream

### ISSUE THREAD

**Issue by vmchale** [haskellari/lukko#18]
```

Failed to build lukko-0.1.1.2.
Build log (
/home/vanessa/.cabal/logs/ghc-8.6.5/lukko-0.1.1.2-0581430e4b3c504ca2ad307ad68211a70aefd87799cd5c70e7cb746c7e6e9558.log
):
Configuring library for lukko-0.1.1.2..
Preprocessing library for lukko-0.1.1.2..
src-ofd/Lukko/OFD.hsc:113 directive let cannot be handled in cross-compilation mode
cabal: Failed to build lukko-0.1.1.2 (which is required by
cabal-install-3.2.0.0). See the build log above for details.
```

Since `lukko` cannot be cross-compiled, trying to cross-compile `cabal-install` with the default flags fails out of the box.

**Comment by phadej** [comment]
How I can reproduce this? Most likely you want `lukko -ofd-locking` constraint, as OFD module shouldn't be built anywhere else than targeting Linux.

**Comment by phadej** [comment]
Would using built-in `#alignment` (when available) fix issue, or are there others? https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/utils.html#input-syntax

**Comment by vmchale** [comment]
Thank you! `-ofd-locking` does indeed work :)

Using `#alignment` would enable cross-compilation I believe. Only let, const_str and def are unsupported: https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/utils.html#cross-compilation

**Comment by phadej** [comment]
Fixed in #21
